src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl

```diff
@@ -111,6 +111,34 @@
     </xsl:if>
   </xsl:template>
 
+  <xsl:template match="note">
+    <xsl:variable name="type">
```

Comment:
As I mentioned earlier, this is not DFM but an XML comment.  The spec uses lowercase.